var/spack/repos/builtin/packages/py-crossmap/package.py

```diff
@@ -13,6 +13,7 @@ class PyCrossmap(PythonPackage, SourceforgePackage):
     homepage = "http://crossmap.sourceforge.net/"
     sourceforge_mirror_path = "crossmap/CrossMap-0.3.3.tar.gz"
 
+    version('0.3.9', sha256='e20a4653e9fc313ac0f5a6cfc37b42e83c3cf2b42f9483706cfb9ec9ff72c74c')
     version('0.3.3', sha256='56d99fd606e13e399b83438953d0d89fc281df1c1e8e47eed7d773e7ec9c88f8')
     version('0.2.9', sha256='57243ee5051352c93088874c797ceac0426f249704ba897360fb628b3365d0af')
 
```

Comment:
I don't see where numpy is required. Also, you should specify that `cython@0.17:` is required